src/sentry/conf/server.py

```diff
@@ -923,6 +923,10 @@ def create_partitioned_queues(name):
 SENTRY_PROJECT = 1
 SENTRY_PROJECT_KEY = None
 
+# Default organization to represent the Internal Sentry project.
+# Used as a default when in SINGLE_ORGANIZATION mode.
+SENTRY_ORGANIZATION = 1
```

Comment:
It'd be nice if this ended with `_ID` but I guess this is at least consistent with the other settings.

Kind of weird that this gets set and potentially used if we're not in single organization mode.